.github/workflows/airgap-test.yml

```diff
@@ -0,0 +1,44 @@
+name: Airgap
+on:
+  - push
+  - pull_request
+
+# Declare default permissions as read only.
+permissions: read-all
+
+jobs:
+  airgap-e2e-test:
+    name: Test Suite
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2.5.0
+      - name: Run registry
+        run: |
+          export CONTAINER_ID=$(docker run -d -p 5000:5000 --name registry registry:2) 
+          echo "CONTAINER_ID=${CONTAINER_ID}" >> $GITHUB_ENV
+      - name: Install kwctl
+        run: cargo install --path .
+      - name: Save policies
+        run: ./scripts/kubewarden-save-policies.sh --policies-list e2e-tests/airgap/policies.txt --policies policies.tar.gz
+      - name: Remove policies from store
+        run: |
+          kwctl rm registry://ghcr.io/kubewarden/policies/capabilities-psp:v0.1.9
```

Comment:
Please, don't use an official policy. This would influence our usage stats

Reply:
yes, you are totally right! done in https://github.com/kubewarden/kwctl/pull/354/commits/43440e53145a57611ac80f1651c50d9bb982704a